test/harnesses/EverlongTest.sol

```diff
@@ -57,7 +57,7 @@ contract EverlongTest is HyperdriveTest, IEverlongEvents {
     int256 internal VARIABLE_RATE = 0.10e18;
 
     uint256 internal INITIAL_VAULT_SHARE_PRICE = 1e18;
-    uint256 internal INITIAL_CONTRIBUTION = 500_000_000e18;
+    uint256 internal INITIAL_CONTRIBUTION = 2_000_000e18;
 
     uint256 internal CURVE_FEE = 0.01e18;
     uint256 internal FLAT_FEE = 0.0005e18;
```

Comment:
This comment is meant for `function setUp`, but it isn't a part of the diff. We should just be able to remove that function. 

```
function setUp() public virtual override {
    super.setUp();
}
```

doesn't actually override anything.